packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -136,6 +136,7 @@ const renderHeaderContent = ({
   startTime,
   timezone,
   locale,
+  offscreenLiveText,
```

Comment:
I think the `offscreenLiveText` is the same as the `stateLabel`, why don't we use that instead?

Reply:
Changed it:)